client/src/styles/global.scss

```diff
@@ -1,2 +1,31 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
+  width: 100%;
+  padding-left: 50%;
+}
+
+.j40Footer {
+  font-family: "sans-serif";
+  font-size: 12px;
+  line-height: 24px;
+}
+
+.j40FooterAddressTitle {
+  line-height: 2.5;
+  font-weight: bold;
+  white-space: nowrap;
+}
+
+.j40FooterLogo {
+  max-width: 120px;
+}
+
+// no way to control style of line address line items via react+trussworks so we're
+// overriding the style used.
+.usa-footer__contact-info {
```

Comment:
Is it possible to start with the default options here and check with design?

Reply:
Sure. All the style tweaks were mostly to see how much work it was to match the design as closely as possible with minimal css adjustments.

Reply:
Yeah I think for now we can start with what comes out of the box, then tweak post-convo with design ?